integrationtesting/opendcs-tests/build.gradle

```diff
@@ -34,6 +39,10 @@ test {
     def classPath = (project(':opendcs').jar.outputs.files
                   + project(":opendcs").configurations.runtimeClasspath).asPath
     inputs.property "opendcs.test.engine", project.findProperty("opendcs.test.engine")
+
+    if (System.getProperty("OpenDCSAlgoTestFilter") != null) {
```

Comment:
follow the concept  on line 47 below. property name should be lower case and start with `opendcs.` e.g. `opendcs.test.algorithm.filter`